Utilities/bootstrap

```diff
@@ -270,6 +270,10 @@ class Target(object):
         if args.llbuild_link_framework:
             other_args.extend(["-F", args.llbuild_build_dir])
 
+        # Don't use GNU strerror_r on Android.
+        if 'ANDROID_DATA' in os.environ:
+            other_args.extend(["-Xcc", "-U_GNU_SOURCE"])
```

Comment:
I am trying to move SwiftPM away from this bootstrap script and rely the CMake 3.15.1 which has builtin Swift support. I don't know when that'll happen but I expect to it to happen within next couple of months (ofc not a guarantee). So this might mean you have to do some of these changes again. If you want to try the new WIP CMake bootstrapping, run `Utilities/new-bootstrap`.

Reply:
Sounds good, I'm going to try cross-compiling SPM itself from linux to Android next, so I was going to try the CMake support for that.